internal/rollup/terser-wrapped.js

```diff
@@ -20,47 +20,81 @@
 // each .js file in that folder and outputs to a specified output
 // folder. If the input is a file then terser is just run
 // one that individual file.
-
 const fs = require('fs');
 const path = require('path');
-const tmp = require('tmp');
-const child_process = require('child_process');
 
-const DEBUG = false;
+// worker_threads is only avilable on version > 10.5.0
+const [major, minor] = process.version.replace('v', '').split('.');
+const WORKER_THREADS_AVILABLE = major > 10 || major == 10 && minor > 5
+let worker_threads;
 
-// capture the inputs and output options
-const argv = require('minimist')(process.argv.slice(2));
-const inputs = argv._;
-const output = argv.output || argv.o;
-const debug = argv.debug;
-const configFile = argv['config-file'];
-// delete the properties extracted above as the remaining
-// arguments are forwarded to terser in execFileSync below
-delete argv._;
-delete argv.output;
-delete argv.o;
-delete argv.debug;
-delete argv['config-file'];
-
-if (DEBUG)
-  console.error(`
-terser: running with
-  cwd: ${process.cwd()}
-  argv: ${process.argv.slice(2).join(' ')}
-  inputs: ${JSON.stringify(inputs)}
-  output: ${output}
-  debug: ${debug}
-`);
-
-if (inputs.length != 1) {
-  throw new Error(`Only one input file supported: ${inputs}`);
+if (WORKER_THREADS_AVILABLE) {
+  worker_threads = require('worker_threads');
+} else {
+  console.warn(
+      `WARNING: worker_threads not are not avilable on your version of Nodejs: ${process.version}
+    Running Terser in serial mode
+    Upgrade your Nodejs version to 10.5.0 minimum for worker_threads to be avilable`)
```

Comment:
we should be more specific here, to update the /WORKSPACE file, otherwise users will update the nodejs on their machine and it still won't work